Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -117,5 +148,21 @@ public void OnInputClicked(InputEventData eventData)
                 anchorManager.AttachAnchor(gameObject, SavedAnchorFriendlyName);
             }
         }
+
+        private void DetermineParent()
+        {
+            if (ParentGameObjectToPlace == null)
+            {
+                if (gameObject.transform.parent == null)
+                {
+                    Debug.LogError("The selected GameObject has no parent.");
+                    PlaceParentOnTap = false;
+                }
+                else
+                {
+                    ParentGameObjectToPlace = gameObject.transform.parent.gameObject;
```

Comment:
@S-Hodgson-MMOARgames On line 154 we check to see if ParentGameObjectToPlace was specified in the inspector. Only if it has no been specified do we try to figure out the parent on our own.